apps/heureka-legacy/src/hooks/useUrlState.ts

```diff
@@ -4,8 +4,7 @@
  */
 
 import { useLayoutEffect, useEffect, useState } from "react"
-// @ts-expect-error TS(2792): Cannot find module '@cloudoperators/juno-url-state... Remove this comment to see the full error message
-import { registerConsumer } from "@cloudoperators/juno-url-state-provider-v1"
```

Comment:
We can leave the application as it is or? we will remove it soonish...

Reply:
Technically `juno-url-state-provider-v1` is not different that `juno-url-state-provider` after this change and because we're going to get rid of this dependency it makes sense to change here as well. In fact I think we should remove `heureka-legacy` altogether we can recover it any time from the commit history if needed.